packages/mui-joy/src/index.ts

```diff
@@ -36,6 +36,8 @@ export * from './Breadcrumbs';
 export { default as Button } from './Button';
 export * from './Button';
 
+export * from './className';
```

Comment:
I wonder if we want to export this as it contains internal className utilities. This will increase the bundle size minutely. cc @siriwatknp